src/org/ods/OdsContext.groovy

```diff
@@ -312,6 +312,10 @@ class OdsContext implements Context {
       config.sonarQubeBranch
   }
 
+  void setSonarQubeBranch(String sonarQubeBranch) {
```

Comment:
We also need this in the interface, `Context.groovy`.

Reply:
@michaelsauter  - worked without ... reason?

Reply:
fixed